src/type/__tests__/enumType.js

```diff
@@ -30,6 +30,13 @@ describe('Type System: Enum Values', () => {
     }
   });
 
+  var FavoriteType = new GraphQLObjectType({
```

Comment:
I tried that too and it worked. I only created this `FavoriteType` to more closely match the problematic case I had in my original code. I can remove it if you prefer (but in my opinion, the more complex the case, the better).